### DISCUSSION TITLE
jurnalfaktual.id - Tak bisa di scroll kebawah sampai mentok

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1595] [General]
Kembali ke atas saat di scroll sampai bawah
![Screenshot_2023-10-18-20-12-34-49_4d38fce200f96aeac5e860e739312e76.jpg](https://github.com/realodix/AdBlockID/assets/114223791/1e4e8028-caaa-4fb0-9f88-80b90fd9cf3f)

https://jurnalfaktual.id/rupa-rupa/sosmed-an/nonton-film-jepang-tanpa-vpn-begini-caranya-dengan-yandex-browser-di-safari/

**Comment by 3xploiton3** [comment]
tampilan chrome desktop aman

![image](https://github.com/realodix/AdBlockID/assets/19517680/da5d83ae-75ca-4314-84de-f9d71ec04e54)

@realodix tambahkan filter cookies kang
